code/modules/modular_computers/file_system/programs/command/card.dm

```diff
@@ -193,6 +221,57 @@
 				else if(href_list["epswd"])
 					var/email_password = input("Enter email password.", "Email password")
 					id_card.associated_email_login["password"] = email_password
+				else if(href_list["sex"])
+					var/sex = input("Type gender.", "Gender")
+					if(!isnull(sex))
+						id_card.sex = sex
+				else if(href_list["age"])
+					var/sug_age = text2num(input("Enter age.", "Age", id_card.age))
+					if(!isnull(sug_age))
+						id_card.age = sug_age
+				else if(href_list["fingerprint_hash"])
+					var/sug_fingerprint_hash = sanitize(input("Enter fingerprint hash.", "Fingerprint hash", id_card.fingerprint_hash), 32)
+					if(hash_check.Find(sug_fingerprint_hash))
+						id_card.fingerprint_hash = sug_fingerprint_hash
+				else if(href_list["dna_hash"])
+					var/sug_dna_hash = sanitize(input("Enter DNA hash.", "DNA hash", id_card.dna_hash), 32)
+					if(hash_check.Find(sug_dna_hash))
+						id_card.dna_hash = sug_dna_hash
+				else if(href_list["blood_type"])
+					var/sug_blood_type = input("Select blood type.", "Blood type") as null|anything in GLOB.blood_types
```

Comment:
Need to do a `CanUseTopic`-type check after every `input` call in this whole proc.